Assets/Scripts/FeedingSystem/DogFoodBowl.cs

```diff
@@ -0,0 +1,77 @@
+﻿/*
+ * Author: Timothy Ng
+ * Description: Handles the feeding dog code through calling FeedDogs
+ */
+
+using UnityEngine.UI;
+
```

Comment:
Hey Tim, I know now we're just getting nitpicky (unfortunately our codebase / conventions are like that). But there should only be a single newline between the `using` directives and the class header, not two